prijateli_tree/app/routers/administration.py

```diff
@@ -171,8 +174,11 @@ def create_game(
     pos_players = [pos_one, pos_two, pos_three, pos_four, pos_five, pos_six]
 
     if len(set(pos_players)) != 6:
+        redirect_url = URL("/admin/game").include_query_params(
+            error="A game cannot contain the same student at two different positions."
+        )
```

Comment:
It's kind of hacky, but without having some sort of `flash` functionality like Flask, this feels like the best move.